schemas/groups/bilge.json

```diff
@@ -0,0 +1,90 @@
+{
+    "type": "object",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "id": "https://signalk.org/specification/1.0.0/schemas/groups/bilge.json#",
+    "description": "A bilge, named by a unique identifier",
+    "properties": {
+        "pump": {
+            "type": "object",
+            "description": "The specific pump of the named bilge, e.g., main, auxiliary, emergency",
+            "properties": {
+                "power": {
+                    "type": "object",
+                    "description": "Current state of electric power available to this pump: ON or OFF",
+                    "allOf": [
+                        {
+                            "$ref": "../definitions.json#/definitions/commonValueFields"
+                        },
+                        {
+                            "properties": {
+                                "value": {
+                                    "enum": [
+                                        "ON",
+                                        "OFF"
+                                    ]
+                                }
+                            }
+                        }
+                    ]
+                },
+                "state": {
+                    "type": "object",
+                    "description": "Current state of this pump: ON or OFF",
+                    "allOf": [
+                        {
+                            "$ref": "../definitions.json#/definitions/commonValueFields"
+                        },
+                        {
+                            "properties": {
+                                "value": {
+                                    "enum": [
+                                        "ON",
+                                        "OFF"
+                                    ]
+                                }
+                            }
+                        }
+                    ]
+                },
+                "counter": {
```

Comment:
Here was my thinking. I might program SensESP to continually send the runCount in the last 3 hours, since for me, that's a number that's a good baseline to determine if there's a problem. So in my case, it would absolutely be "in last X hours". And the path would be something like /bilge/engineRoom/auxiliary/runCount. It would not be /bilge/engineRoom/auxiliary/runCountInLast3Hours. For something like a bilge pump, I would think that "in a certain time period" is the most meaningful number. What does a runCount of 26 mean if I haven't reset since last year?